src/testers.jl

```diff
@@ -251,7 +251,9 @@ function rrule_test(f, ȳ, xx̄s::Tuple{Any, Any}...; rtol=1e-9, atol=1e-9, fdm
     xs = first.(xx̄s)
     accumulated_x̄ = last.(xx̄s)
     check_inferred && _test_inferred(rrule, f, xs...; fkwargs...)
-    y_ad, pullback = rrule(f, xs...; fkwargs...)
+    res = rrule(f, xs...; fkwargs...)
+    isnothing(res) && throw("no rrule defined for $f, with arguments $(typeof(xs))")
```

Comment:
What do you think if we throw a `MethodError` hear instead.
Just so it gives the list of near candidates?

Reply:
Anything better than `iterate(nothing)` is fine, really.

Reply:
New changed.